signer/src/network/in_memory.rs

```diff
@@ -84,8 +95,9 @@ impl super::MessageTransfer for MpmcBroadcaster {
         let mut msg = self.receiver.recv().await.map_err(Error::ChannelReceive)?;
         tracing::trace!("[network{:0>2}] received: {}", self.id, msg);
 
-        while Some(&msg.id()) == self.recently_sent.front() {
-            self.recently_sent.pop_front();
+        //let mut recently_sent = ;
```

Comment:
Forgot this

Reply:
:see_no_evil: removed in [c6110e9](https://github.com/stacks-network/sbtc/pull/670/commits/c6110e91f1ba53184468bc91c6e5e3e30edbe599)